### PR TITLE
adding User.find_or_create_from_access_token_request for token based authorization

### PR DESCRIPTION
**Pull request by perryqh**
adding User.find_or_create_from_access_token_request for token based authorization

### CODE REVIEW

**Comment by perryqh** [comment]
The build has been failing for a while with `The command "gem install bundler" failed and exited with 1 during .
`

File: app/models/g5_authenticatable/user.rb [hunk 1]
```diff
@@ -44,9 +45,7 @@ def locations
       G5Updatable::LocationPolicy::Scope.new(self, G5Updatable::Location)
                                         .resolve
     end
-
-    private
-
```
Comment (mmitchellg5) [inline app/models/g5_authenticatable/user.rb]: did you mean to remove the `private` — dont see `add_scoped_role` being called anywhere else

File: lib/g5_authenticatable/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module G5Authenticatable
-  VERSION = '1.1.4'
+  VERSION = '1.1.5-1'
```
Comment (mmitchellg5) [inline lib/g5_authenticatable/version.rb]: is this a pre version?

Comment (ryanmcadler) [reply lib/g5_authenticatable/version.rb]: It currently is...thinking we should push out a minor release when this is accepted?

**Comment by jcstringer** [comment]
Looks we have some CI failures on this open source gem, not sure if anybody has been keeping up with TravisCI on this one. Appears to have been failing for many months so maybe we can call it good if specs pass locally

https://travis-ci.org/G5/g5_authenticatable/builds?utm_medium=notification&utm_source=github_status

File: app/models/g5_authenticatable/user.rb (hunk 1 above)
Comment (perryqh) [reply app/models/g5_authenticatable/user.rb]: Yep, it's getting used in the new class that was created.

**Comment by perryqh** [comment]
> Looks we have some CI failures on this open source gem, not sure if anybody has been keeping up with TravisCI on this one. Appears to have been failing for many months so maybe we can call it good if specs pass locally
> 
> https://travis-ci.org/G5/g5_authenticatable/builds?utm_medium=notification&utm_source=github_status

@jcstringer yep, I mentioned it above. It seems to be linked to devise_g5_authenticable changes once we get past the bundler errors. I spent a few hours on it without any success and gave up.

File: app/models/g5_authenticatable/user.rb (hunk 1 above)
Comment (mmitchellg5) [reply app/models/g5_authenticatable/user.rb]: 👍

File: lib/g5_authenticatable/version.rb (hunk 2 above)
Comment (mmitchellg5) [reply lib/g5_authenticatable/version.rb]: Yeah we can build 1.1.5 or 1.2.0 when accepted - i'm leaning towards the minor level bump as well

**Pull request merged**